script.js

```diff
@@ -13,6 +13,10 @@ const syncNicknamesButton = document.getElementById(SYNC_NICKNAMES);
 const syncUsersStatusUpdate = document.getElementById(SYNC_USERS_STATUS_UPDATE);
 const syncIdleUsersButton = document.getElementById(SYNC_IDLE_USERS);
 const syncIdleUsersUpdate = document.getElementById(SYNC_IDLE_USERS_UPDATE);
+const syncIdle7dUsersButton = document.getElementById(SYNC_IDLE_7D_Plus_USERS);
```

Comment:
why is this required if we are not using it in code

Reply:
im using it in a function `addClickEventListener` on line number 331 in same file and that SYNC_IDLE_7D_Plus_USERS is coming from `constant.js` file.